.gitignore

```diff
@@ -44,5 +44,5 @@ vue/*
 .terraform
 *.pem
 
-.vscode/*
+.vscode
```

Comment:
Out of curiosity, what's the diff here?

Reply:
New one will look recursively into each directory, old one only looks at the root level

Reply:
nice! sorry, think you may have mentioned this on an older PR and I missed it

Reply:
No worries! It’s pretty unintuitive, I still don’t really understand why it works like that